docs/semgrep-supply-chain/dependency-search.md

```diff
@@ -0,0 +1,87 @@
+---
+slug: dependency-search
+append_help_link: true
+title: Dependency search 
+hide_title: true
+description: "Search through all your dependencies in all your onboarded repositories at any time."
+tags:
+  - Semgrep Supply Chain
+  - Team & Enterprise Tier
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+<ul id="tag__badge-list">
+{
+Object.entries(frontMatter).filter(
+    frontmatter => frontmatter[0] === 'tags')[0].pop().map(
+    (value) => <li class='tag__badge-item'>{value}</li> )
+}
+</ul>
+
+# Searching through your dependencies
+
+Semgrep Supply Chain's dependency search feature allows you to query for any dependency in your codebase at any time. This feature detects all transitive and direct dependencies across all of your repositories in Semgrep Cloud Platform (SCP). Dependency search lists all the versions of a dependency as well as the repositories that use the dependency.
+
+For newly discovered vulnerabilities, which may not yet have a formal CVE or Supply Chain rule, you can use dependency search to discover if you use the vulnerable dependency across all your repositories. You can also use dependency search to see all the versions of a dependency, which can be useful for standardization purposes.
+
+:::info Feature maturity
+Dependency search is in **beta**. To enable this feature, reach out to [andy@semgrep.com](mailto:andy@semgrep.com).
+:::
+
+![Screenshot of default dependency search page](/img/SSC-DepSearch.png)
+*Figure 1.* Default dependency search page.
+
+## Using dependency search
+
+:::info Prerequisites
+* You can only use dependency search through Semgrep Cloud Platform. [Sign up or sign in to Semgrep Cloud Platform](https://semgrep.dev/login).
```

Comment:
Maybe we can use `[Sign up or sign in to Semgrep Cloud Platform](https://semgrep.dev/login)` as a standard in all docs? It looks great! (Feel free to resolve this comment instantly...)